tests/flow/refinements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2738,7 +2738,8 @@ type Breakfast = Apple | Orange | Broccoli | Carrot;
 function bar(x: Breakfast) {
   if (x.kind === "Fruit") {
     (x.taste: "Good");
-  } // error, Apple.taste = Bad
+  }
+  // error, Apple.taste = Bad
```

Comment:
This shouldn't change. As you might have noticed, Prettier distinguishes between 3 kinds of comments: "end of line", "own line", and "remaining" (not included in the first two categories). So, end-of-line comments shouldn't move here.

Reply:
I would favor keeping the comment on its own line in this case as well, since it seems to be less connected to the consequent block than to the alternate.

Reply:
@brodybits Have you never seen code like this?

```js
if (condition) {
  ...
  ... lots of line
  ...
} // if (condition)
```

Same for `while`, `for`, etc. Prettier keeps those comments after `}`.

Anyways, that would be a separate change that would need to be discussed. In this PR, this shouldn't change.